ledger/participant-integration-api/src/main/scala/platform/apiserver/AuthServiceConfig.scala

```diff
@@ -32,7 +32,7 @@ object AuthServiceConfig {
     // exception when reading secret from a config value
     private lazy val verifier =
       HMAC256Verifier(secret, jwtTimestampLeeway).valueOr(err =>
-        throw new IllegalArgumentException(s"Invalid hmac secret ($secret): $err")
+        throw new IllegalArgumentException(s"Failed to create HMAC256 verifier (secret: $secret): $err")
```

Comment:
The message above says it can only throw an exception if secret is null... I am wondering what has changed? maybe comment needs to be updated as well

Reply:
jwtTimestampLeeway can also throw errors when the options are negative.

Reply:
You are right, I will update the comment as well.

Reply:
_will delete the comment since its purpose is a bit unclear_